packages/graphql-typescript-definitions/src/cli.ts

```diff
@@ -47,8 +48,12 @@ const argv = yargs
   })
   .help().argv;
 
+const cwd = process.cwd();
+const graphQLFiles = relative(cwd, argv._[0]);
```

Comment:
we do this so that we can get protect against absolute glob paths.

Reply:
This might actually want a comment in code since it's not very obvious